hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/states/endpoint/HeartbeatEndpointTask.java

```diff
@@ -179,7 +179,7 @@ public EndpointStateMachine.EndPointStates call() throws Exception {
       addQueuedCommandCounts(requestBuilder);
       SCMHeartbeatRequestProto request = requestBuilder.build();
       if (LOG.isDebugEnabled()) {
```

Comment:
If we are changing this, we can remove the `if (LOG.isDebugEnabled()) {` wrapping the debug statement and then change the debug statement to:

```
LOG.debug("Sending heartbeat message : {}", request);
```

Ie, remove the toString call, which will be called by the logging internals only if debug is enabled.

Reply:
Thanks for your comment and review, @sodonnel .
The request here contains a lot of information. Do we need to remove the if judgment?
`
2023-11-15 18:09:34,567 [EndpointStateMachine task thread for xxxx/xxxx.xxxx.xxxx.xxxx:9861 - 0 ] DEBUG org.apache.hadoop.ozone.container.common.states.endpoint.HeartbeatEndpointTask: Sending heartbeat message :: datanodeDetails {
  uuid: "xxxx.xxxx.xxxx.xxxx"
  ipAddress: "xxxx.xxxx.xxxx.xxxx"
  hostName: "xxxxxxxxx"
  ports {
    name: "HTTP"
    value: 9882
  }
  ports {
    name: "CLIENT_RPC"
    value: 9864
  }
  ports {
    name: "REPLICATION"
    value: 9886
  }
  ports {
    name: "RATIS"
    value: 9858
  }
  ports {
    name: "RATIS_ADMIN"
    value: 9857
  }
  ports {
    name: "RATIS_SERVER"
    value: 9856
  }
  ports {
    name: "STANDALONE"
    value: 9859
  }
  networkName: "xxxxxx-xxxx-xxxx-xxxx-xxxxxx"
  networkLocation: "/default-rack"
  persistedOpState: IN_SERVICE
  persistedOpStateExpiry: 0
  uuid128 {
    mostSigBits: xxxxxx
    leastSigBits: xxxxxx
  }
}
dataNodeLayoutVersion {
  metadataLayoutVersion: 7
  softwareLayoutVersion: 7
}
commandQueueReport {
  command: setNodeOperationalStateCommand
  command: replicateContainerCommand
  command: reconstructECContainersCommand
  command: deleteContainerCommand
  command: closeContainerCommand
  command: refreshVolumeUsageInfo
  command: closePipelineCommand
  command: createPipelineCommand
  command: deleteBlocksCommand
  command: finalizeNewLayoutVersionCommand
  count: 0
  count: 0
  count: 0
  count: 0
  count: 0
  count: 0
  count: 0
  count: 0
  count: 0
  count: 0
}
`

Reply:
Inside the LOG.debug method it has the if statement, and the toString will only be called if it need to be logged.

Reply:
OK, I will update soon.